src/QsCompiler/Tests.Compiler/TestCases/LocalVerification.qs

```diff
@@ -51,6 +173,78 @@ namespace Microsoft.Quantum.Testing.LocalVerification {
         using ((q1, q2, q3) = (Qubit(), (Qubit(), Qubit()))) {}
     }
 
+    operation VariableDeclaration11<'T>(cnt: Int, arg : 'T) : Unit {
```

Comment:
Same here, some descriptions would go a long way in being able to tell what the scenario is on each of these.

Reply:
see the response above.